package.json

```diff
@@ -41,5 +41,8 @@
       "pre-commit": "npm run lint",
       "pre-push": "npm run lint && npm test"
     }
+  },
+  "dependencies": {
+    "gcp-jupyterlab-shared": "^1.0.0"
```

Comment:
I'm not sure why this is getting hoisted up to here. Is there another place where it's being referenced in one of the subpackages possibly?

Reply:
Removed it from here and the package-lock.json file, that seems to have worked